apps/gallery/stories/theme/Colors.mdx

```diff
@@ -89,12 +98,12 @@ import { Meta, ColorPalette, Title, ColorItem } from '@storybook/blocks'
 
 <ColorPalette>
   <ColorItem
-    title="Blue"
+    title="Accent"
     subtitle="80 | 90 | 100"
     colors={{
-      'wui-color-blue-80': '#2978CC',
-      'wui-color-blue-90': '#2D7DD2',
-      'wui-color-blue-100': '#3396FF'
+      'wui-accent-080': 'hsla(230, 59%, 55%, 1)',
+      'wui-accent-090': 'hsla(231, 76%, 61%, 1)',
+      'wui-accent-100': 'hsla(230, 100%, 67%, 1)'
```

Comment:
why use hsla here? cannot find a ref to these values in the colors figma (maybe I'm blind?)